docs/demos/AIS_Solent_Tracker.py

```diff
@@ -20,8 +20,9 @@
 # %%
 # Building the detector
 # ---------------------
-# First we will prepare our detector for the |AIS|_ data, which is in CSV file, using the
-# Stone Soup generic CSV reader.
+# First we will prepare our detector for the |AIS|_ data, which is in a
+# :download:`CSV file <../../demos/SolentAIS_20160112_130211.csv>`, using the Stone Soup generic
+# CSV reader.
```

Comment:
While I can see the contents of the csv file, I can't seem to view it in a separate tab, or download and view it. Deliberate?